external_usage/Makefile.in

```diff
@@ -0,0 +1,43 @@
+# makefile template for configuration or runtime replace 
+PARACEL_INSTALL_PREFIX = @PARACEL_INSTALL_PREFIX@
+
+CXX = @CXX@
+CXXFLAGS = -Wall -O3 -std=c++11
+
+CPPFLAGS = -I$(PARACEL_INSTALL_PREFIX)/include
+
+LDFLAGS =  -L$(PARACEL_INSTALL_PREFIX)/lib
+
+LIBS = -lcomm -lscheduler
+LIBS += -lzmq -lmsgpack -lgtest -lgflags -lglog
```

Comment:
Is `-lgtest` necessary for this demo? It will (may) cause linker (ld) error. 
If I removed it, everything works fine.